.gitignore

```diff
@@ -35,6 +35,9 @@ FirebaseSegmentation/Tests/Sample/GoogleService-Info.plist
 # FirebaseMLModelDownloader integration tests GoogleService-Info.plist
 FirebaseMLModelDownloader/Tests/Integration/Resources/GoogleService-Info.plist
 
+# FirebasePerformance dev test App and integration tests GoogleService-Info.plist
+FirebasePerformance/**/GoogleService-Info.plist
```

Comment:
Should we ignore this plist file in the whole repo instead of just FirebasePerformance?

Reply:
+1 @paulb777 What do you think?

Reply:
No. We have some dummy plists in the repo that are sufficient for build only and some unit testing.

Reply:
Make sense. I will ignore the plist only in FirebasePerformance.